code/modules/library/skill_learning/skillchip.dm

```diff
@@ -96,3 +98,23 @@
 	skill_icon = "spa"
 	implanting_message = "<span class='notice'>Your mind is filled with plant arrangments.</span>"
 	removal_message = "<span class='notice'>Your can't remember how a hedge looks like anymore.</span>"
+
+/obj/item/skillchip/useless_adapter
+	name = "Skillchip adapter"
+	skill_name = "Useless adapter"
+	skill_description = "Allows you to insert another identical skillchip into this adapter, but the adapter also takes a slot ..."
+	skill_icon = "plug"
+	implanting_message = "<span class='notice'>You can now implant another chip into this adapter, but the adapter also took up an existing slot ...</span>"
+	removal_message = "<span class='notice'>You no longer have the useless skillchip adapter.</span>"
+	allow_multiple = 1
+	slot_cost = 0
+
+/obj/item/skillchip/useless_adapter/on_apply(mob/living/carbon/user, silent)
+	. = ..()
+	user.adjust_max_skillchip_count_without_updating(1)
+	user.used_skillchip_slots++
+
+/obj/item/skillchip/useless_adapter/on_removal(mob/living/carbon/user, silent)
+	. = ..()
+	user.adjust_max_skillchip_count_without_updating(-1)
+	user.used_skillchip_slots--
```

Comment:
```suggestion
	user.adjust_max_skillchip_count_without_updating(-1)
	user.used_skillchip_slots--
	return ..()
```

Reply:
Changed it to directly return instead of setting return code (in both procs)